client/components/happiness-support/test/index.jsx

```diff
@@ -81,137 +96,129 @@ describe( 'HappinessSupport', () => {
 			recordTracksEvent: noop,
 		};
 
-		beforeEach( () => {
-			wrapper = shallow(
+		test( 'should be rendered only when showLiveChatButton prop is true and LiveChat is available', () => {
+			const { rerender } = render(
 				<HappinessSupport { ...props } showLiveChatButton={ true } liveChatAvailable={ true } />
 			);
-		} );
-
-		test( 'should be rendered only when showLiveChatButton prop is true and LiveChat is available', () => {
 			// should be rendered here
-			expect( wrapper.find( HappychatButton ) ).toHaveLength( 1 );
+			expect( screen.getByTestId( 'happychat-button' ) ).toBeVisible();
 
 			// false cases
-			wrapper = shallow( <HappinessSupport { ...props } /> );
-			expect( wrapper.find( HappychatButton ) ).toHaveLength( 0 );
+			const queryHappychatButton = () => screen.queryByTestId( 'happychat-button' );
+			rerender( <HappinessSupport { ...props } /> );
+			expect( queryHappychatButton() ).not.toBeInTheDocument();
 
-			wrapper = shallow( <HappinessSupport { ...props } showLiveChatButton={ true } /> );
-			expect( wrapper.find( HappychatButton ) ).toHaveLength( 0 );
+			rerender( <HappinessSupport { ...props } showLiveChatButton /> );
+			expect( queryHappychatButton() ).not.toBeInTheDocument();
 
-			wrapper = shallow( <HappinessSupport { ...props } showLiveChatButton={ false } /> );
-			expect( wrapper.find( HappychatButton ) ).toHaveLength( 0 );
+			rerender( <HappinessSupport { ...props } showLiveChatButton={ false } /> );
+			expect( queryHappychatButton() ).not.toBeInTheDocument();
 
-			wrapper = shallow(
-				<HappinessSupport { ...props } showLiveChatButton={ true } liveChatAvailable={ false } />
-			);
-			expect( wrapper.find( HappychatButton ) ).toHaveLength( 0 );
+			rerender( <HappinessSupport { ...props } showLiveChatButton liveChatAvailable={ false } /> );
+			expect( queryHappychatButton() ).not.toBeInTheDocument();
 
-			wrapper = shallow(
-				<HappinessSupport { ...props } showLiveChatButton={ false } liveChatAvailable={ true } />
-			);
-			expect( wrapper.find( HappychatButton ) ).toHaveLength( 0 );
+			rerender( <HappinessSupport { ...props } showLiveChatButton={ false } liveChatAvailable /> );
+			expect( queryHappychatButton() ).not.toBeInTheDocument();
 
-			wrapper = shallow(
+			rerender(
 				<HappinessSupport { ...props } showLiveChatButton={ false } liveChatAvailable={ false } />
 			);
-			expect( wrapper.find( HappychatButton ) ).toHaveLength( 0 );
+			expect( queryHappychatButton() ).not.toBeInTheDocument();
 		} );
 
 		test( 'should render translated content', () => {
-			expect( wrapper.find( HappychatButton ).props().children ).toEqual(
+			render(
+				<HappinessSupport { ...props } showLiveChatButton={ true } liveChatAvailable={ true } />
+			);
+			expect( screen.queryByTestId( 'happychat-button' ) ).toHaveTextContent(
```

Comment:
We could use a `get*` query here since we're querying for an existing element:

```suggestion
			expect( screen.getByTestId( 'happychat-button' ) ).toHaveTextContent(
```